src/scripts/oakitems/OakItemRunner.ts

```diff
@@ -1,97 +1,99 @@
 
         // TODO implement use!
         // TODO implement functionality
-        OakItemRunner.oakItemList.push(ko.observable(new OakItem("Cell Battery", 90, "Regenerate more mining energy", 25, 5, 4)));
+        OakItemRunner.oakItemList.push(ko.observable(new OakItem(GameConstants.OakItem.CellBattery, "Cell Battery", 90, "Regenerate more mining energy", 25, 5, 4)));
 
-        let item: OakItem = OakItemRunner.getOakItemByName("Magic Ball");
+        let item: OakItem = OakItemRunner.getOakItemObject(GameConstants.OakItem.MagicBall);
         OakItemRunner.selectedItem(item);
     }
 
     public static loadOakItems() {
         let oakItems = JSON.parse(JSON.stringify(player._oakItemsEquipped))
         for (let i = 0; i < oakItems.length; i++) {
-            OakItemRunner.activateOakItem(OakItemRunner.getOakItemByName(oakItems[i]).id);
+            OakItemRunner.activateOakItem(OakItemRunner.getOakItemObject(oakItems[i]).id);
         }
         for(let i = 0; i<OakItemRunner.oakItemList.length; i++){
             OakItemRunner.oakItemList[i]().calculateLevel();
         }
     }
 
-    public static hover(name: string) {
-        OakItemRunner.inspectedItem(OakItemRunner.getOakItemByName(name));
+    public static hover(id: GameConstants.OakItem) {
+        OakItemRunner.inspectedItem(OakItemRunner.getOakItemObject(id));
     }
 
     public static hoverRelease() {
         OakItemRunner.inspectedItem(OakItemRunner.selectedItem());
     }
 
-    public static click(name: string) {
-        let item: OakItem = OakItemRunner.getOakItemByName(name);
+    public static click(id: GameConstants.OakItem) {
+        let item: OakItem = OakItemRunner.getOakItemObject(id);
         OakItemRunner.selectedItem(item);
         if(item.isUnlocked()) {
             OakItemRunner.activateOakItem(item.id);
         }
     }
 
-    public static use(name: string) {
-        OakItemRunner.getOakItemByName(name).use();
+    public static use(id: GameConstants.OakItem) {
+        OakItemRunner.getOakItemObject(id).use();
     }
 
-    public static calculateBonus(name: string): number {
-        return OakItemRunner.getOakItemByName(name).calculateBonus()();
+    public static calculateBonus(id: GameConstants.OakItem): number {
+        return OakItemRunner.getOakItemObject(id).calculateBonus()();
     }
 
-    public static getOakItemByName(name: string): OakItem {
-        for (let i = 0; i < OakItemRunner.oakItemList.length; i++) {
-            if (OakItemRunner.oakItemList[i]().name() == name) {
-                return OakItemRunner.oakItemList[i]();
-            }
-        }
+    public static getOakItemObject(id: GameConstants.OakItem): OakItem {
+        // for (let i = 0; i < OakItemRunner.oakItemList.length; i++) {
+        //     if (OakItemRunner.oakItemList[i]().id == id) {
+        //         return OakItemRunner.oakItemList[i]();
+        //     }
+        // }
```

Comment:
Remove commented code

Reply:
Removed. I added a comment to note that `OakItemRunner.oakItemList` must follow the order of the enum.